### PR TITLE
Babel preset.

### PR DESCRIPTION
**Pull request by nobuti**
This PR updates Babel presets.

### CODE REVIEW

File: webpack.dev.config.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ module.exports = env => {
             resolve(__dirname, 'node_modules/tangram.cartodb')
           ],
           options: {
-            presets: ['es2015']
+            presets: ['env']
```
Comment (ivanmalagon) [inline webpack.dev.config.js]: I'd add the browsers configuration to be sure that the transpilation targets our requirements.

Something like
```
"targets": {
          "browsers": ["last 2 versions", "ie >= 11"]
        }
```

File: webpack.prod.config.js [hunk 2]
```diff
@@ -102,7 +102,7 @@ module.exports = env => {
             resolve(__dirname, 'node_modules/tangram.cartodb')
           ],
           options: {
-            presets: ['es2015']
+            presets: ['env']
```
Comment (ivanmalagon) [inline webpack.prod.config.js]: Same here.

File: webpack.dev.config.js (hunk 1 above)
Comment (nobuti) [reply webpack.dev.config.js]: But our browsers requirements seems pretty complex: https://github.com/CartoDB/cartodb#requirements, it's pretty much the same as not having any target.

Comment (ivanmalagon) [reply webpack.dev.config.js]: But if you don't set any target, the plugin tries to be the most restrictive or the least? That's my doubt.

Comment (nobuti) [reply webpack.dev.config.js]: From the docs: 

> The default behavior without options runs all transforms. 

> Without any configuration options, babel-preset-env behaves exactly the same as babel-preset-latest (or babel-preset-es2015, babel-preset-es2016, and babel-preset-es2017 together).

So, it'd include:
- es2017
- es2016
- es2015

**Comment by nobuti** [comment]
Retest this please

File: webpack.dev.config.js (hunk 1 above)
Comment (ivanmalagon) [reply webpack.dev.config.js]: Ok then. Transpile all tings!

**Pull request closed without merging**